sirepo/uri_router.py

```diff
@@ -116,7 +116,7 @@ def init(app, simulation_db):
 
     for n in _REQUIRED_MODULES + tuple(sorted(feature_config.cfg().api_modules)):
         register_api_module(importlib.import_module('sirepo.' + n))
-    _init_uris(app, simulation_db)
+    _init_uris(app, simulation_db, feature_config.cfg().sim_types)
```

Comment:
no need to complicate this interface by passing sim_types. Rather either pass feature_config or nothing and have _init_uris import feature_config, which can be imported more easily than simulation_db, which has import loop issues.

Reply:
https://github.com/radiasoft/sirepo/pull/2786/commits/1912882dc322db1f33962e87148aeddc9c6ef2d7